src/envoy/http/jwt_auth/http_filter.h

```diff
@@ -51,6 +51,8 @@ class JwtVerificationFilter : public StreamDecoderFilter,
   StreamDecoderFilterCallbacks* decoder_callbacks_;
   // The auth object.
   JwtAuth::JwtAuthenticator jwt_auth_;
+  // The auth store object, which can be used to get the config
```

Comment:
try to keep only what you need. It avoid any unintentional uses and make it easy to know the purpose of each variables. In this case, you can just need one boolean, or at most, ref to the config.